managed/src/main/java/com/yugabyte/yw/commissioner/tasks/DestroyUniverse.java

```diff
@@ -55,6 +66,53 @@ public void run() {
         universe = lockUniverseForUpdate(-1 , true);
       }
 
+      if (params().isDeleteBackups) {
+        List<Backup> backupList = Backup.fetchByUniverseUUID(params().customerUUID, universe.universeUUID);
+        for (Backup backup : backupList) {
+          try {
```

Comment:
This part of the code is almost same as the delete backup task. Let me know where I can move this and use the common function in both files.

Reply:
@Arnav15

Reply:
Hmm, @mahendranbhat do you think we can make this a subtask? That way we can use the same subtask in both tasks. Let me know if that makes sense?

Reply:
we have to raise an exception in the backup task but here we should not if delete backup failed for any reason ( I assume, correct me if I am wrong ). So in the subtask, one more param to identify whether the subtask can raise an exception or not? Or can we keep it like this only?

Reply:
Yes @mahendranbhat , that is correct. Though, I think a force delete task ignores subtask failures anyway, so you might not have to build it in the subtask. The actual task framework should take care of it. So basically you will create a new subtask, and you can actually put them in the same subtask group and run all of them together. The task framework already has a mechanism to ignore errors if one subtask in the group fails.